### PR TITLE
Feature: Remove remaining mocks

### PR DESCRIPTION
**Pull request by josemarinas**
## Description

Remove the remaining mocks in the sdk

Task: [APP-1401](https://aragonassociation.atlassian.net/browse/APP-1401)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder of the package after the [UPCOMING] title and before the latest version.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: modules/client/test/integration/client/estimation.test.ts [hunk 1]
```diff
@@ -37,7 +40,17 @@ describe("Client", () => {
...
         metadataUri: `ipfs://QmeJ4kRW21RRgjywi9ydvY44kfx71x2WbRq7ik5xh5zBZK`,
         ensSubdomain: daoName.toLowerCase().replace(" ", "-"),
-        plugins: [{ id: "0x1234", data: new Uint8Array() }],
+        plugins: [
+          {
+            id: deployment.addressListRepo.address,
+            data: toUtf8Bytes(
+              defaultAbiCoder.encode(
+                ["uint64", "uint64", "uint64", "address[]"],
+                [1, 1, 1, []],
```
Comment (brickpop) [inline modules/client/test/integration/client/estimation.test.ts:49]: I guess these `1, 1, 1` don't alter the estimation, but don't we have the real values?

File: modules/client/test/integration/client/methods.test.ts [hunk 2]
```diff
@@ -68,41 +68,6 @@ describe("Client", () => {
     });
 
     describe("DAO Creation", () => {
-      it("Should estimate gas fees for creating a DAO", async () => {
-        const context = new Context(contextParamsLocalChain);
-        const client = new Client(context);
-
-        const daoName = "ERC20VotingDAO_" +
-          Math.floor(Random.getFloat() * 9999) + 1;
```
Comment (brickpop) [inline modules/client/test/integration/client/methods.test.ts:76]: Why are we removing this test, especially now?

File: modules/client/test/integration/client/estimation.test.ts (hunk 1 above)
Comment (josemarinas) [reply modules/client/test/integration/client/estimation.test.ts:49]: Sure, I can change it to the real values

File: modules/client/test/integration/client/methods.test.ts (hunk 2 above)
Comment (josemarinas) [reply modules/client/test/integration/client/methods.test.ts:76]: Is the same test, it was duplicated

**Review by brickpop — APPROVED** [review]

**Pull request merged**